### ISSUE TITLE
Insufficient color contrast of code samples in PDF documentation

### ISSUE THREAD

**Issue by vytas7** [falconry/falcon#1249]
As reported on chat by @xakiy, code samples in documentation are hard to read due to very light text on white background when rendered in PDF format. A similar issue (https://github.com/falconry/falcon/issues/1214) has been resolved in 4eb5384 for HTML version by changing Pygments code style to ``monokai``. However, for some reason ``monokai`` dark background does not follow through to LaTeX output which is later used to render PDF.

**Comment by edmondb** [comment]
I think I can tackle this one...

**Comment by edmondb** [comment]
Question @nZac, as of #1330 this was changed to `github` style rather than `monokai`. Which is preferred for this as I believe this negates #1214 ...

**Comment by edmondb** [comment]
As an update as I need to pick this up at a later time...

- __Note:__ The `github` style used with Pygments in the documentation configuration was something that was required and installed via `pip install pigments-style-github`.
- Trying to compile the LaTeX using `make latex` then `make latexpdf` gives me a Unicode error that is irrelevant to this issue in the first place. That unicode character [(U+2518)](https://www.fileformat.info/info/unicode/char/2518/index.htm) is used in some of the verbatim fields for code examples and diagrams such a directory trees. One can correct this by adding the line `latex_engine = 'xelatex'` in the conf.py file at line 258.
- Once the character is fixed, one is able to redefine the verbatim environment in LaTeX, but this is not an easy task. I've tried redefining this command, environment, etc. It's actually the field `sphinxVerbatim` in the .tex file instead, but to change the background of those environments, one will need to add something here:

```python
latex_elements = {
    'preamble': r'''
\usepackage{xcolor}
\let\oldv\sphinxVerbatim
\let\oldendv\endsphinxVerbatim

\def\sphinxVerbatim{\par\setbox0\vbox\bgroup\oldv}
\def\endsphinxVerbatim{\oldendv\egroup\fboxsep0pti \noindent\colorbox[gray]{0.8}{\usebox0}\par}
''',

This gives a grey background __behind__ the verbatim field instead of changing the field itself. Also, to note, this would not do well for verbatim that spans across page breaks.

**Comment by vytas7** [comment]
Closing this due to the lack of our bandwidth to investigate this, and also because it seems that the latest docs on RtD don't really suffer from this issue: https://falcon.readthedocs.io/_/downloads/en/latest/pdf/
![image](https://user-images.githubusercontent.com/3430939/227714533-23d9f24e-3639-404b-a19d-9d1f0dd9a388.png)

**Comment by vytas7** [comment]
I have explicitly selected a high contrast, most black-and-white theme for the Latex version as part of adopting the PyData theme in Falcon 4.0. Resolved fixed.
